### ISSUE TITLE
ESLint server does not work

### ISSUE THREAD

**Issue by LionyxML** [emacs-lsp/lsp-mode#3916]
### Thank you for the bug report

- [X] I am using the latest version of `lsp-mode` related packages.
- [X] I checked [FAQ](https://emacs-lsp.github.io/lsp-mode/page/faq/) and [Troubleshooting](https://emacs-lsp.github.io/lsp-mode/page/troubleshooting/) sections
- [X] You may also try reproduce the issue using clean environment using the following command: `M-x lsp-start-plain`

### Bug description

Strangely  `lsp-eslint.el` have the variable `lsp-eslint-server-command` pointing to `~/server/out/eslintServer.js`.

How is this file supposed to be there? Also, ~ at this point is just `myprojectPath/~...` which is also wrong.

The only way I got ESLint working was following this post from 6 months ago:
https://www.reddit.com/r/emacs/comments/wdqjxu/lspeslintservercommand_not_working_anymore/

Basically, installing globally: `npm install -g vscode-langservers-extracted`

And setting the command with: 
`(setq lsp-eslint-server-command '("vscode-eslint-language-server" "--stdio"))`

 Now everything is ok and ESLint is working.

### Steps to reproduce

Open a file such as App.tsx of this repo: 

### Expected behavior

Invoking lsp should start ts-ls and eslint.

Only ts-ls is loaded.

### Which Language Server did you use?

ts-ls
eslint

### OS

MacOS

**Comment by LionyxML** [comment]
And magically nothing works again :( Sorry I'll keep digging... but no success anymore
